pages/linux/nl.md

```diff
@@ -0,0 +1,24 @@
+# nl
+
+> A utility for numbering lines, either from a file, or from standard input.
+
+- Number lines in a file:
+
+`nl {{source_file}}`
+
+- Number lines where there is printable text only:
+
+`nl -t {{source_file}}`
+
+- Number lines that match a regular expression (Starting with a Capital 
+A is Exampled):
```

Comment:
Please don't manually break lines. Also, would `-p '^A'` work? If so I think it would make it clearer (and perhaps "regular expression" --> "regex pattern" would allow the connection to the actual option letter.)